### ISSUE TITLE
Add method get_hyperparams in ddpg, vpg

### ISSUE THREAD

**Issue by Zeus3101** [SforAiDl/JigglypuffRL#60]
When render is set as True, there is an error with saving.
The main error seems to be:
`ValueError: ctypes objects containing pointers cannot be pickled `

It is likely that the env.render options saves some ctype objects containing pointers.

`  File "\anaconda3\envs\torch\lib\site-packages\torch\serialization.py", line 328, in save
    _legacy_save(obj, opened_file, pickle_module, pickle_protocol)
  File "\anaconda3\envs\torch\lib\site-packages\torch\serialization.py", line 401, in _legacy_save
    pickler.dump(obj)
ValueError: ctypes objects containing pointers cannot be pickled
Exception ignored in: <function Viewer.__del__ at 0x000001721BB77D30>
Traceback (most recent call last):
  File "\anaconda3\envs\torch\lib\site-packages\gym\envs\classic_control\rendering.py", line 152, in __del__
  File "\anaconda3\envs\torch\lib\site-packages\gym\envs\classic_control\rendering.py", line 71, in close
  File "\anaconda3\envs\torch\lib\site-packages\pyglet\window\win32\__init__.py", line 299, in close
  File "\anaconda3\envs\torch\lib\site-packages\pyglet\window\__init__.py", line 820, in close
ImportError: sys.meta_path is None, Python is likely shutting down`

**Comment by Sharad24** [comment]
This is probably happening because we save `env` as well. We'd want to save only the hyperparams.
We should probably create a method `get_hyperparams` in every Algo

**Comment by Zeus3101** [comment]
Should I change save_params so that env is not saved? Also, I can work on the the get_hyperparams function. Let me know which hyperparams would be the important ones.

**Comment by Sharad24** [comment]
Ideally having a separate function which returns important hyperparams sounds better to me.
Something like:
```
class Algo():
    def get_hyperparams(self):
        hyperparams = {'lr':self.lr, 'actor_network':self.ac.state_dict()}
        return hyperparams

**Comment by ajaysub110** [comment]
Closing this. Will be fixed in #79
